website/content/api-docs/libraries.mdx

```diff
@@ -51,7 +51,10 @@ $ pip install ansible-modules-hashivault
 
 ### C&#35;
 
-- [VaultSharp](https://github.com/rajanadar/VaultSharp) (.NET Standard 2.0 & 1.3, .NET Core >= 1.0.0, .NET 4.5, .NET 4.6, Mono, iOS, Android, Mac, UWP etc.)
+- [VaultSharp](https://github.com/rajanadar/VaultSharp) (The most comprehensive cross-platform .NET Library for Vault) 
+  - Supports all Auth and Secret backends supported by Vault including Enterprise features of Vault.
+  - Truly cross platform: Supports .NET 5, .NET 6, .NET Framework 4.x, .NET Standard 2.x/1.x, mac/iOS, Android, UWP, Mono, Xamarin etc.
+  - With extensive documentation and code samples at https://rajanadar.github.io/VaultSharp
```

Comment:
```suggestion
- [VaultSharp](https://github.com/rajanadar/VaultSharp)
  - Supports .NET 5, .NET 6, .NET Framework 4.x, .NET Standard 2.x/1.x, mac/iOS, Android, UWP, Mono, Xamarin.
```

Let's keep these to a minimum to match the other descriptions. Make sure to keep your readme updated with the correct information.

Reply:
> Let's keep these to a minimum to match the other descriptions. Make sure to keep your readme updated with the correct information.

got it. made the change